library/Rules/DateTimeDiff.php

```diff
@@ -36,22 +35,20 @@ final class DateTimeDiff extends Standard
     private readonly Validatable $rule;
 
     /** 
-     * @param string $type "years"|"months"|"days"|"hours"|"minutes"|"seconds"|"microseconds" 
-     * @param string|null $format Example: "Y-m-d H:i:s.u"
+     * @param string $type DateInterval format examples: (y, m, d, days, h, i, s, f)
```

Comment:
I do understand the appeal of using DateInterval formats directly. However, I think having more descriptive values is better as it's more understandable. With the literal constant, there's not reason to translate anything.